tests/example_experiments_test.py

```diff
@@ -0,0 +1,95 @@
+# pylint: disable=invalid-name,no-self-use
+import pyhocon
+
+from deep_qa.common.params import Params, replace_none
+from deep_qa.models import concrete_models
+
+from .common.test_case import DeepQaTestCase
+
+
+class TestExampleExperiments(DeepQaTestCase):
+    def setUp(self):
+        super(TestExampleExperiments, self).setUp()
+        self.example_experiments_dir = "./example_experiments"
```

Comment:
Not sure this setup is buying us much at this point, as each experiment file is tested individually now.  It'd probably be simpler to just give the full path in a test directly.  Then it's easier to think about adding new ones, and checking if one is there.  Fewer places to modify or look at when you're adding a new test.